meta/rpc.go

```diff
@@ -351,6 +398,29 @@ func (r *rpc) join(localAddr, remoteAddr string) (*JoinResult, error) {
 	}
 }
 
+// promoteToRaft attempts to promote a node at remoteAddr using localAddr as the current
+// node's cluster address
+func (r *rpc) promoteToRaft(addr string, peers []string) error {
```

Comment:
Should we refer to this as `enableRaft` since `state.go` uses that naming?  That way the code doesn't have references to promotion and enable in different spots.  Might avoid some confusion in the future.